netbox/utilities/api.py

```diff
@@ -95,23 +97,37 @@ def get_prefetches_for_serializer(serializer_class, fields_to_include=None):
         if serializer_field and serializer_field.source:
             model_field_name = serializer_field.source
 
+        # If the serializer field is a RelatedObjectCountField and its a nested field
+        # Add an annotation to the annotaded_prefetch
+        if isinstance(serializer_field, RelatedObjectCountField) and source_field is not None:
+            if model_field_name not in annotaded_prefetch:
+                annotaded_prefetch[model_field_name] = Count(serializer_field.relation)
+
         # If the serializer field does not map to a discrete model field, skip it.
         try:
             field = model._meta.get_field(model_field_name)
-            if isinstance(field, (RelatedField, ManyToOneRel, GenericForeignKey)):
+            if (isinstance(field, (RelatedField, ManyToOneRel, GenericForeignKey)) and
```

Comment:
This can be moved below the try/except - if the field doesn't exist we are doing a continue so it won't get hit and field will be set.
Also - if it is `issubclass(type(serializer_field), Serializer)` do we want to continue to line 117?  Not sure - just asking, could we just do a check on that condition and continue?